web_src/css/repo.css

```diff
@@ -2195,18 +2195,12 @@ td .commit-summary {
   display: inline-flex;
   flex-wrap: wrap;
   gap: 2.5px;
-}
-
-.labels-list a {
-  display: flex;
-  text-decoration: none;
+  align-items: center;
 }
 
 .labels-list .label {
   padding: 0 6px;
-  margin: 0 !important;
   min-height: 20px;
-  display: inline-flex !important;
```

Comment:
Did you check this causes no regression elsewhere, like on issue list?

Reply:
![image](https://github.com/go-gitea/gitea/assets/2114189/8c047552-84a8-4537-b62e-bb57b2fc2dc9)